### PR TITLE
[ICDS Dashboard]: Fixes in the tabular reports

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
##### SUMMARY

- Set the default locationIndex to state
captured (4)

captured (5)

### CODE REVIEW

**Review by Rohit25negi — APPROVED** [review]
Looks good.

File: custom/icds_reports/static/icds_reports/js/spec/month-filter.directive.spec.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ describe('Month Filter Controller', function () {
         injectPPDController();
 
         var result = controller.getPlaceholder();
-        var expected = 'Apr-Jun 2019';
+        var expected = 'Oct-Dec 1969';
```
Comment (Rohit25negi) [inline custom/icds_reports/static/icds_reports/js/spec/month-filter.directive.spec.js:97]: does not the change looks wrong? like above today is in July so the returned quarter should be April-June only no?

**Review by Rohit25negi — CHANGES_REQUESTED** [review]

File: custom/icds_reports/static/icds_reports/js/spec/month-filter.directive.spec.js (hunk 1 above)
Comment (ShubhamBansal1997) [reply custom/icds_reports/static/icds_reports/js/spec/month-filter.directive.spec.js]: updated and reverted!

**Comment by ShubhamBansal1997** [comment]
Build Passed https://travis-ci.org/github/dimagi/commcare-hq/builds/705393760

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**